test/e2e/apimachinery/flowcontrol.go

```diff
@@ -265,22 +265,24 @@ func getPriorityLevelConcurrency(f *framework.Framework, priorityLevelName strin
 
 // createFlowSchema creates a flow schema referring to a particular priority
 // level and matching the username provided.
-func createFlowSchema(f *framework.Framework, flowSchemaName string, matchingPrecedence int32, priorityLevelName string, matchingUsername string) (*flowcontrol.FlowSchema, func()) {
+func createFlowSchema(f *framework.Framework, flowSchemaName string, matchingPrecedence int32, priorityLevelName string, matchingUsernames ...string) (*flowcontrol.FlowSchema, func()) {
```

Comment:
can we make this a `[]string` instead? it feels more natural to me

Reply:
It's a relatively common way of not breaking existing callers, but this file is short enough that it might be just as easy to fix all the callers?

Reply:
yep, there should only be two callers, both in this file

Reply:
update with `[]string`